engine/table/src/test/java/io/deephaven/engine/table/impl/TestAggBy.java

```diff
@@ -341,6 +341,21 @@ public void testAggCountWhere() {
         assertEquals(6L, counts.get(0));
         counts = ColumnVectors.ofLong(doubleCounted, "filter15");
         assertEquals(6L, counts.get(0));
+
+        // Get a static set table for use in dynamic where filters (contains 0-3)
+        final QueryTable setTable = (QueryTable) TableTools.newTable(col("sym", 1, 2, 3));
```

Comment:
Noticed that AggCountWhere didn't test `DynamicWhereFilter` in CI, corrected here.